.coveragerc

```diff
@@ -0,0 +1,16 @@
+[report]
```

Comment:
@engjames Any reason you have this inclusion here. It is used to configure coverage to ignore certain files and you are not working on such a story.

Reply:
Sorted